examples/src/main/java/io/numaproj/numaflow/examples/function/evenodd/Forward.java

```diff
@@ -1,22 +1,22 @@
 package io.numaproj.numaflow.examples.function.evenodd;
 
+import io.numaproj.numaflow.function.Datum;
 import io.numaproj.numaflow.function.FunctionServer;
 import io.numaproj.numaflow.function.Message;
 import io.numaproj.numaflow.function.map.MapFunc;
-import io.numaproj.numaflow.function.v1.Udfunction;
 
 import java.io.IOException;
 import java.util.logging.Logger;
 
 public class Forward {
     private static final Logger logger = Logger.getLogger(Forward.class.getName());
 
-    private static Message[] process(String key, Udfunction.Datum data) {
-        int value = Integer.parseInt(new String(data.getValue().toByteArray()));
+    private static Message[] process(String key, Datum data) {
+        int value = Integer.parseInt(new String(data.getValue()));
```

Comment:
Not introduced by this PR: we are missing exception handling in this example. parseInt can throw and we should catch and handle it gracefully.

Reply:
done